chain/consensus/tendermint/README.md

```diff
@@ -180,3 +179,20 @@ stop-app1
 start-app1
 start-node1
 ```
+
+## Testing
+
+### Sanity Tests
+
+The following command run Eudico consensus sanity testsuite:
+```
+make eudico-test
+```
+
+### Acceptance Tests
+ The following acceptance test can be used to make sure that the Tendermint consensus integration satisfy to the Eudico consensus requirements:
+ 1. Verify messages (funds) can be sent between accounts.
```

Comment:
I love this!! We should maybe make it the baseline tests any consensus implementation needs to at least pass (maybe we can add something else, but as a baseline it is perfect)